javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/matcher/GlobalIgnoresMatcher.java

```diff
@@ -64,6 +64,10 @@ private GlobalIgnoresMatcher(boolean skipAdditionalLibraryMatcher) {
   public boolean matches(T target) {
     String name = target.getActualName();
 
+    if (name.startsWith("jdk.internal.net.http.")) {
```

Comment:
Move this near the check for `java` please.

Reply:
next if checks for `name.startsWith("jdk.")`: 
```
if (name.startsWith("net.bytebuddy.")
        || name.startsWith("jdk.")
        || name.startsWith("org.aspectj.")
        || name.startsWith("com.intellij.rt.debugger.")
        || name.startsWith("com.p6spy.")
        || name.startsWith("com.dynatrace.")
        || name.startsWith("com.jloadtrace.")
        || name.startsWith("com.appdynamics.")
        || name.startsWith("com.singularity.")
        || name.startsWith("com.jinspired.")
        || name.startsWith("org.jinspired.")) {
      return true;
    }
```
That's why I put if before this `if`